sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -807,7 +807,7 @@ public virtual Response<MetricFeedback> GetFeedback(string feedbackId, Cancellat
         /// <summary>
         /// Gets a collection of items describing the anomalies detected by a given <see cref="AnomalyDetectionConfiguration"/>.
         /// </summary>
-        /// <param name="detectionConfigurationId">The unique identifier of the <see cref="MetricAnomalyAlertConfiguration"/>.</param>
+        /// <param name="detectionConfigurationId">The unique identifier of the <see cref="AnomalyAlertConfiguration"/>.</param>
```

Comment:
Noticed that we were referencing the wrong type in the docstrings of this class. Fixing it in this PR.

Reply:
To be fare, names are pretty similar :)